lib/hsa/mcwamp_hsa.cpp

```diff
@@ -339,25 +339,25 @@ class HSAKernel {
                 _hsaExecutableSymbol,
                 HSA_EXECUTABLE_SYMBOL_INFO_KERNEL_GROUP_SEGMENT_SIZE,
                 &this->static_group_segment_size);
-        STATUS_CHECK_Q(status, commandQueue, __LINE__);
+        STATUS_CHECK(status, __LINE__);
```

Comment:
This is a bug.  commandQueue is not defined in HSAKernel.  Somehow it was only exposed on Fedora